scripts/configure.py

```diff
@@ -104,6 +104,7 @@ def configure(build_type):
             "GameManager",
             "Chain",
             "Controller",
+            "CMyFont",
```

Comment:
This should go between Chain and Controller so it ends up in the right location in the executable, I believe?